hs_core/templates/search/search.html

```diff
@@ -145,6 +146,26 @@ <h4 class="text-muted">Temporal Coverage</h4>
                     </table>
                 </form>
                 <br>
+                <div id="resource-search" style="display:none">
```

Comment:
I suggest using the bootstrap class 'hidden' which does the same thing instead of inline style rules.